datasource/threedi_results.py

```diff
@@ -7,74 +7,19 @@
 
 from threedigrid.admin.constants import NO_DATA_VALUE
 
+from .result_constants import layer_object_type_mapping
 from .base import BaseDataSource
-from ..utils import cached_property
```

Comment:
De wijzigingen in deze imports gaan even bijten met mijn imports-veranderd-pull-request.
Even opletten aan beide kanten.